OpenDreamShared/Network/Messages/MsgAlert.cs

```diff
@@ -1,6 +1,6 @@
 ﻿using Lidgren.Network;
 using Robust.Shared.Network;
-
+using Robust.Shared.Serialization;
 namespace OpenDreamShared.Network.Messages
```

Comment:
Leave an empty line between the `using` statements and the `namespace` statement.
```suggestion
using Robust.Shared.Serialization;

namespace OpenDreamShared.Network.Messages
```